queries.sql

```diff
@@ -24,4 +24,8 @@ WHERE name != 'Gabumon';
 
```

Comment:
![image](https://github.com/bilalrajput09/Vet-clinic-database/assets/93039359/87088540-c7c4-451b-856a-81961609d08c)

**[OPTIONAL]**

- [ ] It would be nice if you make your readme more professional by fixing your issues page link in your readme file which is broken 😨 Kindly fix it to make your readme descriptive 👍

Reply:
FIXED !!!